packages/stream/index.d.ts

```diff
@@ -1,46 +1,71 @@
-export * from './types.js';
 import { AsyncQueue, Stream, Reader, Writer } from './types.js';
 
+export * from './types.js';
+
 export function makeQueue<TValue>(): AsyncQueue<TValue>;
 
 export function makeStream<
   TRead,
   TWrite = undefined,
   TReadReturn = undefined,
-  TWriteReturn = undefined,
+  TWriteReturn = undefined
 >(
-  acks: AsyncQueue<IteratorResult<TRead, TReadReturn>>,
-  data: AsyncQueue<IteratorResult<TWrite, TWriteReturn>>,
+  _acks: AsyncQueue<IteratorResult<TRead, TReadReturn>>,
+  _data: AsyncQueue<IteratorResult<TWrite, TWriteReturn>>,
```

Comment:
I disagree with this change. The type definition file should have meaningfully named parameters as they serve the purpose of documentation. Satisfying inapplicable eslint rules shouldn't impact the quality of definitions.